src/graphic/helper/roundSector.ts

```diff
@@ -138,38 +169,36 @@ export function buildPath(ctx: CanvasRenderingContext2D | PathProxy, shape: {
         arc = mathAbs(tmpAngles[0] - tmpAngles[1]);
     }
 
-    const cx = shape.cx;
-    const cy = shape.cy;
-    const cornerRadius = shape.cornerRadius || 0;
-    const innerCornerRadius = shape.innerCornerRadius || 0;
+    const [icrStart, icrEnd, ocrStart, ocrEnd] = normalizeCornerRadius(cornerRadius, innerRadius, radius);
```

Comment:
Could we optimize it a bit and not do this normalizing if `cornerRadius` is not given?

Reply:
Fixed.